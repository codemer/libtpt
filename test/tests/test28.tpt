#!/usr/local/bin/tpt
This test should ignore the first line.
