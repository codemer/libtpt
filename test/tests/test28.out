This test should ignore the first line.
